project-generated-code-backend/Backend/Service/SchedulingService/IRoomRenovationDatabaseSql.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace HealthClinicBackend.Backend.Service.SchedulingService
+{
+    internal interface IRoomRenovationDatabaseSql
+    {
+    }
+}
```

Comment:
I think this was made by mistake

Reply:
I will check this, because of Azure. Last time he complained on the Interfaces of Repositories.